src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -85,47 +86,63 @@ def _call_autofix(
         instruction: str,
         timeout_secs: int,
     ):
-        response = requests.post(
-            f"{settings.SEER_AUTOFIX_URL}/v1/automation/autofix/start",
-            data=orjson.dumps(
-                {
-                    "organization_id": group.organization.id,
-                    "project_id": group.project.id,
-                    "repos": repos,
-                    "issue": {
-                        "id": group.id,
-                        "title": group.title,
-                        "short_id": group.qualified_short_id,
-                        "events": [serialized_event],
-                    },
-                    "instruction": instruction,
-                    "timeout_secs": timeout_secs,
-                    "last_updated": datetime.now().isoformat(),
-                    "invoking_user": (
-                        {
-                            "id": user.id,
-                            "display_name": user.get_display_name(),
-                        }
-                        if not isinstance(user, AnonymousUser)
-                        else None
-                    ),
+        path = "/v1/automation/autofix/start"
+        body = orjson.dumps(
+            {
+                "organization_id": group.organization.id,
+                "project_id": group.project.id,
+                "repos": repos,
+                "issue": {
+                    "id": group.id,
+                    "title": group.title,
+                    "short_id": group.qualified_short_id,
+                    "events": [serialized_event],
                 },
-                option=orjson.OPT_NON_STR_KEYS,
-            ),
-            headers={"content-type": "application/json;charset=utf-8"},
+                "instruction": instruction,
+                "timeout_secs": timeout_secs,
+                "last_updated": datetime.now().isoformat(),
+                "invoking_user": (
+                    {
+                        "id": user.id,
+                        "display_name": user.get_display_name(),
+                    }
+                    if not isinstance(user, AnonymousUser)
+                    else None
+                ),
+            },
+            option=orjson.OPT_NON_STR_KEYS,
+        )
+        response = requests.post(
+            f"{settings.SEER_AUTOFIX_URL}{path}",
+            data=body,
+            headers={
+                "content-type": "application/json;charset=utf-8",
+                **sign_with_seer_secret(
+                    path=path,
+                    body=body,
+                ),
+            },
         )
```

Comment:
Yes, although that is a bit deeper of a change with some larger risks as well.  Not all call sites behave the same way and I don't have full confidence I understand their contexts, in part because those callsites are maintained by other teams.  My first thought was to get this change in safely and then consider other changes.